web/web/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "webpack-dev-server --mode development --config webpack.config.js",
-    "build": "webpack --watch --mode production"
+    "build": "webpack --watch --mode development"
```

Comment:
can you explain this change? Is it just so it re-build automatically?

Reply:
I can change it back - but yes it's to rebuild automatically

Reply:
feel free to add a `build-dev` or `start-dev` or `debug` script that does this though.

Reply:
ok awesome, done